versions.txt

```diff
@@ -7,3 +7,23 @@ cc_shim_version=3.0.0-alpha.3
 clear_vm_kernel_version=4.9.35-76
 clear_vm_image_version=16180
 qemu_lite_version=741f430a960b5b67745670e8270db91aeb083c5f
+
+# OBS package versions
+# Versions matches the ones in the OBS stable release.
+# When making a new release of the runtime, this file
+# should contain the new versions.
+
+# Fedora
+proxy_obs_fedora_version=3.0.0alpha.2+git.7cb0341-1.1
+shim_obs_fedora_version=3.0.0alpha.2+git.9d1a3ee-1.1
+image_obs_fedora_version=16180-30.1
+selinux_obs_fedora_version=0.1-3.1
+linux_container_obs_fedora_version=4.9.35-66.1
+qemu_lite_obs_fedora_version=2.7.1+git.741f430a96-6.1
+
+# Ubuntu
+proxy_obs_ubuntu_version=3.0.0alpha.2+git.7cb0341-1.1
+shim_obs_ubuntu_version=3.0.0alpha.2+git.9d1a3ee-1.1
```

Comment:
shim is version alpha.4 

https://github.com/clearcontainers/shim/tree/3.0.0-alpha.4